JSDemos/Demos/Gauges/GaugeTitleCustomized/React/description.md

```diff
@@ -0,0 +1,11 @@
+This demo shows how to specify the CircularGauge title and display custom labels in the component's center. 
```

Comment:
A sentence like this implies that it teaches you how to specify a title in any CircularGauge component. So I would use "a" instead of "the". 

```suggestion
This demo shows how to specify a CircularGauge title and display custom labels in the component's center. 
```